Editor/Completion/XmlCompletionCommitManager.cs

```diff
@@ -38,10 +38,39 @@ public CommitResult TryCommit (IAsyncCompletionSession session, ITextBuffer buff
 			}
 
 			switch (kind) {
+			case XmlCompletionItemKind.SelfClosingElement: {
+					if (typedChar != '>') {
+						string insertionText = $"{item.InsertText}/>";
+						Insert (session, buffer, insertionText);
+						ShiftCaret (session, 2, XmlCaretDirection.Left);
+						return CommitResult.Handled;
+					} else {
+						return CommitResult.Unhandled;
+					}
+				}
+			case XmlCompletionItemKind.Element: {
+					string insertionText = $"{item.InsertText}></{item.InsertText}>";
+					Insert (session, buffer, insertionText);
+					ShiftCaret (session, item.InsertText.Length + 3, XmlCaretDirection.Left);
+					return CommitResult.Handled;
+				}
+			case XmlCompletionItemKind.Attribute: {
+					string insertionText = $"{item.InsertText}=" + @"""""";
```

Comment:
IMO this would read better as ` $"{item.InsertText}=\"\""`

Reply:
updated